documentation/modules/appendix/pages/best-practices.adoc

```diff
@@ -12,56 +12,854 @@ SPDX-License-Identifier: MPL-2.0
+
+|===
+|Characteristic held by Property|How to name the Property |How to name the Aspect
+
+|xref:ROOT:characteristics.adoc#collection-characteristic[Collection] Characteristic
+|Name the Property `items`.
+|Put the Aspect's name in *plural*.
+
+|xref:ROOT:characteristics.adoc#list-characteristic[List] Characteristic
+|Name the Property `items`.
+|Put the Aspect's name in *plural*.
+
+|xref:ROOT:characteristics.adoc#set-characteristic[Set] Characteristic
+|Name the Property `items`.
+|Put the Aspect's name in *plural*.
+
+|xref:ROOT:characteristics.adoc#sorted-set-characteristic[Sorted Set] Characteristic
+|Name the Property `items`.
+|Put the Aspect's name in *plural*.
+
+|xref:ROOT:characteristics.adoc#time-series-characteristic[Time Series] Characteristic
+|Name the Property `items`.
+|Put the Aspect's name in *singular* and add a `History` suffix to it.
+|===
+
+[[the-preferredname-attribute]]
+== The `preferredName` attribute
+
+The `preferredName` attribute is available for all model elements.
+
+For more information, refer to xref:ROOT:modeling-guidelines.adoc#attributes-that-all-model-elements-have[Attributes that all model elements have].
+
+For optimal results, consider the following best practices:
+
+* <<precise-and-understandable>>
+* <<separate-words>>
+* <<no-article>>
+* <<capitalization-as-in-dictionary>>
+
+In addition to that, there are also best practices for:
+
+* <<the-name-attribute>>
+* <<the-description-attribute>>
+
+[[precise-and-understandable]]
+=== Precise and understandable
+
+The `preferredName` attribute is meant to provide a human-readable name.
+
+Choose a Preferred Name that is precise and easy to understand.
+
+[TIP]
+====
+People might encounter the Preferred Name, for example:
+
+* As a user of an Aspect Model--in documentation generated from the Aspect Model
+* As a user of a software solution--in a user interface of an application that was built based on the Aspect Model
+====
+
+[[separate-words]]
+=== Separate words
+
+Should you copy and paste an element's `name` to use it as the `preferredName`, adapt it accordingly.
+
+For example, if the `name` consists of more than one word, written in camel case, separate it into the respective words.
+
+.Examples of Preferred Names derived from the `name` attributes of the model elements
+[source,turtle,subs="attributes+,+quotes"]
+----
+:plantId a samm:Property ;
+   samm:preferredName "plant ID"@en ;
+
+:machineTmperature a samm:Property ;
+   samm:preferredName "machine temperature"@en ;
+
+:euTaxonomyDisclosureStatement a samm:Property ;
+   samm:preferredName "EU taxonomy disclosure statement"@en ;
+
+:ttlFileName a samm:Property ;
+   samm:preferredName "TTL file name"@en ;
+----
+
+[[no-article]]
+=== No article
+
+While an article and a noun can be combined, they do not automatically _belong_ together.
+
+Therefore, avoid articles in Preferred Names.
+
+|===
+|{nok-small} DON'T |{ok-small} DO
```

Comment:
it would be user friednly to add what the dos adn donots are about, e.g. "preferred name", some chapters are quite long and sometimes I loose control what it is about (becuase I mainly look at the examples ;-)